uk.co.compucorp.civicrm.civihr/api/v3/CiviHR.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * CiviHR.getversion API specification.
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC/API+Architecture+Standards
+ */
+function _civicrm_api3_civi_h_r_getversion_spec(&$spec) {
```

Comment:
no need to define this spec function since I don't think we are gonna use it for this API end point.